bazel/extension_config/extensions_build_config.bzl

```diff
@@ -356,17 +345,17 @@ ENVOY_CONTRIB_EXTENSIONS = {
 }
 
 
-ISTIO_DISABLED_EXTENSIONS = [ 
+ISTIO_DISABLED_EXTENSIONS = [
     # ISTIO disable tcp_stats by default because this plugin must be built and running on kernel >= 4.6
     "envoy.transport_sockets.tcp_stats",
```

Comment:
Maybe this can be removed since it was removed above. Not sure if leaving it here will cause a build issue.

Reply:
It's ok to keep or delete.